imports/api/entries/server/publications.js

```diff
@@ -1,4 +1,4 @@
-import { Entries } from '../../lib/collections/entries';
+import Entries from '/imports/api/entries/entries.js';
 
 Meteor.publish('entries.board', function(argument) {
```

Comment:
Can we increase to more than 19, here? (Wondering if we want "scroll down for more entries", here)

Reply:
Yeah why not, but for scrolling behaviour we should consider dynamic loading

Reply:
++